# Number of reaching points
reaching_num: 3

# Regularization weight
reg_weight: 1e-6

# QP objective weight for placement
placement_weight: 1e-3

# QP objective weight for SVM inequality error
svm_ineq_weight: 1e6

# Rate in runLoop()
loop_rate: 5000

# Step interval to publish in runLoop()
publish_interval: 100

# Height of xy plane marker
svm_thre: -0.1

# Limit of configuration update in one step [m], [rad]
delta_config_limit: 0.005

# Whether to predict on grid map
grid_map_prediction: false

# Name of body to reach
ik_body_name: tool0

# Name list of joints which is not used in IK
ik_exclude_joint_name_list: {}

# Number of IK trial
ik_trial_num: 10

# Number of IK loop
ik_loop_num: 50

# Threshold of IK [m], [rad]
ik_error_thre: 2e-3

# Duration between adjacent target poses for animation [s]
animate_adjacent_duration: 1.0

# Division number between adjacent target poses for animation
animate_adjacent_divide_num: 100

# Number of IK loop for animation
animate_ik_loop_num: 5
